plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/rest/TestTrinoRestCatalog.java

```diff
@@ -61,13 +62,15 @@ protected TrinoCatalog createTrinoCatalog(boolean useUniqueTableLocations)
         return new TrinoRestCatalog(restSessionCatalog, new CatalogName(catalogName), NONE, "test", useUniqueTableLocations);
     }
 
+    @Test
```

Comment:
Unrelated to this PR - do we have any plans to figure out "forgotten" test overrides which are missing now `@Test` ?

Reply:
there is automated check for that now.